build_functions/c_util_suppressions.sup

```diff
@@ -18,4 +18,12 @@
    Helgrind:Race
    fun:logger_deinit
    ...
+}
+{
+   <logger_log gets flagged for "Conditional jump or move depends on uninitialised value(s)">
+   Memcheck:Cond
+   fun:__vfprintf_internal
+   fun:__vsnprintf_internal
+   fun:log_sink_console_log
+   fun:logger_log
 }
```

Comment:
This looks a bit fishy to me. How come that this was not flagged up to now and it only pops up now?

Reply:
I think this one we should look into, disabling memcheck for all logging sounds opening up too much for issues cropping in.

Reply:
Removed suppression and got tests to pass using thandle_helper